tests/tests_include_vars_from_parent.yml

```diff
@@ -1,5 +1,15 @@
 - hosts: all
   tasks:
+    - name: Ensure ansible_facts used by role
+      setup:
+        gather_subset:
+          - '!all'
+          - '!any'
+          - os_family
+          - distribution
+          - distribution_version
+          - distribution_major_version
```

Comment:
?  https://docs.ansible.com/ansible/latest/collections/ansible/builtin/setup_module.html#setup-module - I guess `!all` and `!any` should be under `gather_subset`  - but the names of individual facts should be in `filter` - and why is this needed here?

Reply:
Sorry, I thought the test failed without this task, but I cannot reproduce it any more... There should have been some other bug, then...   I'm removing this setup task.